packages/shared/src/lib/featureManagement.ts

```diff
@@ -61,4 +61,17 @@ export const featureFeedAdTemplate = new Feature('feed_ad_template', {
   },
 } as Record<FeedSettingsKeys, FeedAdTemplate>);
 
+export const featureValidLanguages = new Feature('validLanguages', {
```

Comment:
This will just mean we have to maintain 2 right?
One in remote config and one here, are we ok with that?